torch/_dynamo/skipfiles.py

```diff
@@ -224,6 +227,7 @@ def _module_dir(m: types.ModuleType):
 if torch.distributed.is_available():
     MOD_INLINELIST.add("torch.distributed")
     MOD_INLINELIST.add("torch.distributed._functional_collectives")
+    MOD_INLINELIST.add("torch.distributed._composable.replicate")
```

Comment:
@yanboliang are we making a new behavior for dynamo that should remove this list in the future? or are we just going to have to keep using this?

Reply:
This line is legit. The `LEGACY_MOD_INLINELIST` one is the one should be removed according to @yanboliang. But right now we have to add `LEGACY_MOD_INLINELIST` to make it work.

Reply:
yea, that's what i meant. I was asking Yanbo if we are gonna have a new solution for LEGACY_... or if the fact we're still adding to it is a problem